projects/demo/src/app/demo/pages/form-control-page/value-example/form-control-value-example.component.ts

```diff
@@ -0,0 +1,55 @@
+import { ChangeDetectorRef, Component, QueryList, ViewChildren } from '@angular/core';
+import { PaFormControlDirective } from '../../../../../../../pastanaga-angular/src/lib/controls/form-field/pa-form-control.directive';
+import { detectChanges } from '../../../../../../../pastanaga-angular/src';
+import { FormControl } from '@angular/forms';
+
+@Component({
+    selector: 'pa-demo-form-control-value-example',
+    templateUrl: './form-control-value-example.component.html',
+    styleUrls: ['./form-control-value-example.component.scss'],
+})
+export class FormControlValueExampleComponent {
+    standaloneParentValue?: any;
+    standaloneChangeEvent?: any;
+    standaloneCode = `<div paFormControl
+    [value]="..."
+    (valueChange)="..."></div>`;
+    ngModelCode = `<div paFormControl [(ngModel)]="..."></div>`;
+    model = 'initial value';
+    formControl = new FormControl('');
+    formControlCode = `<input paFormControl [formControl]="formControl">`;
+    @ViewChildren(PaFormControlDirective) directives?: QueryList<PaFormControlDirective>;
+
+    constructor(private cdr: ChangeDetectorRef) {}
+
+    onStandaloneChange(event: any) {
+        this.standaloneChangeEvent = event;
+    }
+
+    standaloneParentToChildChange() {
+        // reset former value;
+        this.standaloneParentValue = undefined;
+        detectChanges(this.cdr);
+        this.standaloneParentValue = 'change from parent';
+        detectChanges(this.cdr);
```

Comment:
couldn't you use `markForCheck` instead of `detectChanges` here?

Reply:
I could of course, but since detectChanges is the fastest 
and we don't risk an "expression change after it was checked" here, it felt like the thing to use...

Reply:
detectChanges is not the fastest, it is meant to be used when something happens outside of ngZone. So by default you should always use markForCheck, and keep detectChanges for the few cases where it is actually needed (like in a timeout for example)